resources/css/labki-import-footer.css

```
@@ -0,0 +1,54 @@
+.labki-import-footer {
+  margin-top: 1.75rem;
+  padding: 14px 20px;
+  border: 1px solid #d8dde3;
+  border-left: 4px solid #36c;
+  border-radius: 4px;
+  background: linear-gradient(180deg, #fbfcfe 0%, #f5f6f8 100%);
+  font-size: 0.9rem;
+  color: #4a4f54;
+  display: flex;
+  gap: 0.75rem;
+  align-items: flex-start;
```

```suggestion
  align-items: center;
```

text floats up to the top currently